cmds/webboot/webboot.go

```diff
@@ -128,23 +128,23 @@ func main() {
 		verbose = log.Printf
 	}
 
-	cachedDir := *dir
+	entries := []menu.Entry{
+		&DownloadOption{},
+	}
 
-	if cachedDir == "" {
+	if *dir == "" {
 		mp, err := getCachedDirectory()
-		if err != nil {
-			log.Fatalf("Fail to find the USB stick: %+v", err)
+		if err == nil {
```

Comment:
Might be good to log the error here so that if someone was expecting to see a cached directory but doesn't, they can see what went wrong

Reply:
ok, will revise it in the next PR.